README.md

```diff
@@ -1,3 +1,50 @@
 # eslint-config-stripes
 
-Stripes's .eslintrc as an extensible shared config
+This package provides an extensible shared [ESLint](https://eslint.org) config, intended to promote consistent code style among applications built with [FOLIO Stripes](https://github.com/folio-org/stripes-core).
+
+## Installation
+If you haven't already installed ESLint as a dev dependency in your project, it's a required peer of `eslint-config-stripes`:
+```
+yarn add eslint -D
+```
+
+Then add `eslint-config-stripes` to your `devDependencies`:
+```
+yarn add eslint-config-stripes -D
+```
+
+Create `.eslintrc` in the root of your project. Its contents:
+```
+{
+  "extends": "stripes"
+}
+```
+
+### Optional
+If you're extensively using newer JavaScript syntax with Babel, you may want to use [`babel-eslint`](https://github.com/babel/babel-eslint).
```

Comment:
Is this most cases? It seems like 95% of projects use babel. If so, I'd say "recommended" instead of optional.... or "Using Babel?"

Reply:
Switched to "Using Babel?"